toast-er.html

```diff
@@ -261,6 +288,10 @@
         }
       },
 
+      _buttonTap: function(event) {
+        this.fire("toast-er-" + event.target.buttonEvent);
```

Comment:
@LM450N It's kinda against spec to have a different event prefix to the HTML tag name... I think it's much easier to remember, otherwise you should just rename the element to `toaster`.